README.md

```diff
@@ -49,14 +49,30 @@ from fetchai import fetch
 
 # Your AI's query that it wants to find another
 # AI to help it take action on.
-query = "Buy me a pair of shoes"
-
-# Find the top AIs that can assist your AI with
-# taking real world action on the request.
-available_ais = fetch.ai(query)
-
-print(f"{available_ais.get('ais')}")
-# [
+def search_shoe_agents():
+    """Search for available shoe shopping agents"""
+    try:
+        logger.info("Searching for shoe shopping agents...")
+        
+        # Specific query for shoe shopping agents
+        query = "Buy me a pair of shoes"
+        
+        available_ais = fetch.ai(query)
+    	agents = available_ais.get("agents", [])
```

Comment:
@rajashekarcs2023, this was fixed in the latest version. Please review https://github.com/fetchai/fetchai/pull/34